src/main/java/frc/trigon/robot/subsystems/roller/toohardroller/ToohardRollerConstants.java

```diff
@@ -0,0 +1,50 @@
+package frc.trigon.robot.subsystems.roller.toohardroller;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+public class ToohardRollerConstants {
+    private static final int
+            ANGLE_MOTOR_ID = 0,
+            COLLECTION_MOTOR_ID = 1,
+            FORWARD_LIMIT_SWITCH_CHANNEL = 0,
+            BACKWARD_LIMIT_SWITCH_CHANNEL = 1;
+
+    private static final boolean
+            ANGLE_MOTOR_INVERTED = false,
+            COLLECTION_MOTOR_INVERTED = false;
+    private static final NeutralMode ANGLE_MOTOR_NEUTRAL_MODE = NeutralMode.Brake;
+    private static final CANSparkMax.IdleMode COLLECTION_MOTOR_IDLE_MODE = CANSparkMax.IdleMode.kCoast;
+    private static final int
+            ANGLE_VOLTAGE_COMPENSATION_SATURATION = 12,
+            COLLECTION_VOLTAGE_COMPENSATION_SATURATION = 12;
+    static final TalonSRX ANGLE_MOTOR = new TalonSRX(ANGLE_MOTOR_ID);
+    static final CANSparkMax COLLECTION_MOTOR = new CANSparkMax(COLLECTION_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+    public static final DigitalInput FORWARD_LIMIT_SWITCH = new DigitalInput(FORWARD_LIMIT_SWITCH_CHANNEL);
+    public static final DigitalInput BACKWARD_LIMIT_SWITCH = new DigitalInput(BACKWARD_LIMIT_SWITCH_CHANNEL);
```

Comment:
By definition, these should only be used in the ToohardRollerIO, so these should not be public 
Also, there is no need for the redundant `public static final DigitalInput`